test/InfoPanel_test.jsx

```diff
@@ -12,9 +12,11 @@ describe("InfoPanel", () => {
     assert(panel.hasClass("custom"));
   });
 
-  it("renders title", () => {
+  it("renders title in the Collapsible trigger", () => {
+    const title = <div>Panel Title</div>;
     const panel = shallow(<InfoPanel title="Panel Title" />);
-    assert.equal(panel.find(`.${cssClass.TITLE}`).text(), "Panel Title");
+    assert.deepEqual(panel.find(`.${cssClass.COLLAPSIBLE}`).props().trigger,
+      title);
   });
 
   it("renders content", () => {
```

Comment:
I'm thinking it would be good to do another test here like `it("renders no content when defaultOpen is false")`.  I think it would look just like this test but include a prop of `defaultOpen={true}`, and then `assert.equal` that the `.panel.find()` returns nothing.  At least I'm pretty sure that's the way to test this case.  Happy to pair if it becomes tricky!